CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+
```

Comment:
Is the extra newline here intentional?

Reply:
Yeah it's what I do for the Python releases (eg https://github.com/heroku/heroku-buildpack-python/commit/c9504ffd2ebbfd6a4651948af7c50ddbd7924544) since it means the next change post release only has to add one line, and not additional newlines, eg:
https://github.com/heroku/heroku-buildpack-python/commit/452443d420fa67e531a28c63571b8db979e3433d#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR5

Reply:
Cool, yeah, nice 👍